CRM/Utils/JSON.php

```diff
@@ -39,16 +39,28 @@
 class CRM_Utils_JSON {
 
   /**
-   * Function to create JSON object
+   * Output json to the client.
+   * @param mixed $input
+   */
+  public static function output($input) {
+    header('Content-Type: application/json');
+    CRM_Utils_JSON::output($input);
```

Comment:
found it - I ported this

https://github.com/civicrm/civicrm-core/commit/ecdef330eb57462e49d99c3b0c4e1c976c0e494a
but not the fix for it :-)